pages/linux/debchange.md

```diff
@@ -0,0 +1,16 @@
+# debchange
+
+> Tool for maintenance of the debian/changelog file in a Debian source package.
+> More information: <https://manpages.debian.org/buster/devscripts/debchange.1.en.html>.
+
+- Add a new version for a non-maintainer upload to the changelog:
+
+`debchange --nmu`
```

Comment:
How does it know which source package to add the new version for? Is it the current directory or something?

Sorry, not familiar enough with Debian source packages to know lol

Reply:
Yes, it is based on the current directory. This tool is used from inside the package source folder.

Reply:
Right, I see. Thanks for clarifying :-)